packages/paste-core/components/select/stories/select.stories.tsx

```diff
@@ -726,3 +728,89 @@ export const SelectOverflowLongValue = (): React.ReactNode => {
 SelectOverflowLongValue.story = {
   name: 'Select - overflow long value',
 };
+
+export const CustomizedSelect = (): React.ReactNode => {
+  const currentTheme = useTheme();
+
+  const seed = useUIDSeed();
+
+  return (
+    <CustomizationProvider
+      theme={{
+        ...currentTheme,
+        shadows: {
+          ...currentTheme.shadows,
+          shadowFocus: '0 0 0 4px rgba(3, 154, 165, 0.9)',
+        },
+      }}
+      elements={{
+        SELECT_ELEMENT: {
+          color: 'colorTextNew',
+          variants: {
+            inverse: {
+              color: 'colorTextWarningStrong',
+              fontWeight: 'fontWeightBold',
+            },
+          },
+        },
+        CUSTOM_SELECT_ELEMENT: {
+          color: 'colorTextSuccess',
+          fontWeight: 'fontWeightSemibold',
+          fontFamily: 'fontFamilyCode',
+          variants: {
+            inverse: {
+              color: 'colorTextErrorLight',
+            },
+          },
+        },
+        SELECT_WRAPPER: {
+          boxShadow: 'shadowBorderPrimary',
+          ':hover': {boxShadow: 'shadowBorderError'},
+          ':focus-within': {boxShadow: 'shadowFocus'},
+        },
+        SELECT_ICON_WRAPPER: {
+          transform: 'rotate(90deg) translateX(-50%) translateY(-20%)',
+        },
+        CUSTOM_SELECT_ICON: {
+          color: 'colorTextInverseWeak',
+        },
+      }}
+    >
+      <Box maxWidth="size40" paddingX="space40" paddingY="space80">
```

Comment:
FYI I didn't create a component to share between the tests and stories since I did not want to create scope creep from modifying the unit tests.